App/Views/FileSyncListWindow.xaml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<winuiex:WindowEx
```

Comment:
Not for this PR, but should we consider converting our other windows to WinUIEx?

Reply:
In this case I'm only using it for minimum window size setting. We might start using it in other places, but probably as necessary rather than as a global thing. `WindowEx` inherits `Window` so it's pretty much a drop-in replacement when required